### ISSUE TITLE
fin run-cli composer uses composer 1.10.7 in fin 1.102.0

### ISSUE THREAD

**Issue by davekopecek** [docksal/docksal#1448]
The run-cli command in v1.102.0 uses composer 1.10.7, composer has been updated to 2.0.6 for `fin composer`

**Steps to reproduce the issue:**

1. Run `fin composer -V`  
2. Run `fin run-cli composer -V`

**Describe the results you received:**

```
docksal@DocksalNode ~/sandbox/test
master└─ $ fin composer -V
Composer version 2.0.6 2020-11-07 11:21:17
docksal@DocksalNode ~/sandbox/test
master└─ $ fin run-cli composer -V
Composer version 1.10.7 2020-06-03 10:03:56
```

**Describe the results you expected:**

Expected that `fin run-cli composer` would use Composer 2.x version.

**Comment by shelane** [comment]
The `run-cli` is still set to 2.11 and Composer 2 was added in 2.12.

**Comment by lmakarov** [comment]
Fixed in #1463
